src/main/java/MinesweeperController.java

```diff
@@ -123,14 +124,13 @@ private void placeMines() {
     public boolean isOnGrid (int x, int y){
         int minVal = 0;
         int maxVal = boardSize - 1;
-        if ( x < minVal || x > maxVal || y < minVal ||  y > maxVal){
-            return false;
-        }
-        return true;
+        return x >= minVal && x <= maxVal && y >= minVal && y <= maxVal;
```

Comment:
smart, thanks

Reply:
minVal seems a bit overkill since it's just 0.
maxVal isn't necessary if you do < instead of <=